### PR TITLE
fix hardcoded s3 creds

### PR DESCRIPTION
**Pull request by pkit**
use a node or config based creds when not supplied explicitly

fixes: #75

### CODE REVIEW

File: charts/core-dump-handler/README.md [hunk 1]
```diff
@@ -228,3 +228,38 @@ Daemonset
 * s3BucketName : Maps to the S3_BUCKET_NAME enviroment variable
 * 3Region : Maps to the S3_REGION enviroment variable
 * extraEnvVars: Option for passing additional configuration to the agent such as endpoint properties.
+
+### EKS setup for gitops pipelines (`eksctl` or similar)
+
+Set up a service account with a role that has access to S3 bucket (in `cluster.yanl`):
```
Comment (No9) [inline charts/core-dump-handler/README.md:234]: nit: update file extension to `.yaml`

File: charts/core-dump-handler/README.md [hunk 2]
```diff
@@ -228,3 +228,38 @@ Daemonset
 * s3BucketName : Maps to the S3_BUCKET_NAME enviroment variable
 * 3Region : Maps to the S3_REGION enviroment variable
 * extraEnvVars: Option for passing additional configuration to the agent such as endpoint properties.
+
+### EKS setup for gitops pipelines (`eksctl` or similar)
```
Comment (No9) [inline charts/core-dump-handler/README.md:232]: Can you move this section above the environment section
https://github.com/IBM/core-dump-handler/pull/76/files/5cf0b0b02e9cce21502aa594536cb951f6c5a5ef#diff-35330df803a4bbfce6b8b734490d5f0b77bb117755aeac1958500a4934a86bb8R104

File: charts/core-dump-handler/README.md [hunk 3]
```diff
@@ -228,3 +228,38 @@ Daemonset
...
+      "Effect": "Allow",
+      "Action": "s3:*",
+      "Resource": [
+        "arn:aws:s3:::my-core-dump-bucket",
+        "arn:aws:s3:::my-core-dump-bucket/*"
+      ]
+    }
+  ]
+}
+```
```
Comment (No9) [inline charts/core-dump-handler/README.md:265]: Can you also add a sample command you used to install the chart.
Want to capture how it works with the schema validation

**Review by No9 — CHANGES_REQUESTED** [review]
Added some comments in the README but the code LGTM

File: charts/core-dump-handler/README.md (hunk 3 above)
Comment (pkit) [reply charts/core-dump-handler/README.md]: I'm not using the chart.
It doesn't fit our gitops-only paradigm.
I render the chart offline using `helm template` and then install it through `fluxcd2`

**Comment by No9** [comment]
Thanks again for this I'll merge this PR and sort the fmt issue as part of the release.

**Pull request merged**